packages/gatsby/src/schema/create-key.js

```diff
@@ -1,3 +1,3 @@
 const regex = new RegExp(`[^a-zA-Z0-9_]`, `g`)
 
-module.exports = key => key.replace(regex, `___`)
+module.exports = key => key.replace(`@`, ``).replace(regex, `___`)
```

Comment:
I will cherry-pick your commit 😄  For instant I just try to have a functionnal gatsby env, I will clean this PR before remove `[WIP]` tag 😉